rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -458,8 +458,12 @@ void getActiveFederatorPublicKeyOfType() {
 
         @Test
         void getRetiringFederation() {
-            when(federationSupport.getRetiringFederation()).thenReturn(federation);
-            assertThat(bridgeSupport.getRetiringFederation(), is(federation));
+            Optional<Federation> expectedRetiringFederation = Optional.of(federation);
+            when(federationSupport.getRetiringFederation()).thenReturn(expectedRetiringFederation);
+
+            Optional<Federation> actualRetiringFederation = bridgeSupport.getRetiringFederation();
+            assertTrue(actualRetiringFederation.isPresent());
+            assertThat(actualRetiringFederation.get(), is(federation));
```

Comment:
Why not equals? I think you are comparing instances rather than the federation's content.

Reply:
I would leave it as it is. All tests in that class use the same assertion form